packages/esm-forms-workspace-app/src/forms-workspace/form-list/forms-list.component.tsx

```diff
@@ -0,0 +1,182 @@
+      : formsInfo,
+    pageSize,
+  );
+  const handleFormOpen = useCallback(
+    (formUuid, encounterUuid, formName) => {
+      launchFormEntryOrHtmlForms(visit, formUuid, patient, htmlFormEntryForms, encounterUuid, formName);
+    },
+    [htmlFormEntryForms, patient, visit],
+  );
+  const headers = [
+    {
+      id: 0,
+      header: t('formName', 'Form name (A-Z)'),
+      key: 'formName',
+    },
+    {
+      id: 1,
+      header: t('lastCompleted', 'Last completed'),
+      key: 'lastCompleted',
+      maxWidth: '40%',
+    },
+  ];
+
+  useEffect(() => {
+    const entriesToDisplay = isEmpty(searchTerm)
+      ? formsInfo
+      : formsInfo.filter((formInfo) => {
+          return formInfo.form.name.toLowerCase().search(searchTerm?.toLowerCase()) !== -1;
+        });
+    setFormsInfo(entriesToDisplay);
+  }, [formsInfo, searchTerm]);
+
+  const tableRows: Array<DataTableRow> = useMemo(
+    () =>
+      results?.map((formInfo) => {
+        return {
+          id: formInfo.form.uuid,
+          lastCompleted: formInfo.lastCompleted ? formatDatetime(formInfo.lastCompleted) : undefined,
+          formName: formInfo.form.display ?? formInfo.form.name,
+          formUuid: formInfo.form.uuid,
+          encounterUuid: formInfo?.associatedEncounters[0]?.uuid,
+        };
+      }),
+    [results],
+  );
+
+  if (tableRows) {
+    return (
+      <div className={styles.formsListContainer}>
+        <CardHeader title={t(formsSection.labelCode, formsSection.name)} children={undefined} />
+        <div className={styles.container}>
+          <DataTable size={isDesktop(layout) ? 'sm' : 'lg'} rows={tableRows} headers={headers} isSortable>
+            {({ rows, headers, getHeaderProps, getRowProps, getTableProps }) => (
+              <TableContainer className={styles.tableContainer}>
+                <Table {...getTableProps()} useZebraStyles={true}>
+                  <TableHead>
+                    <TableRow>
+                      {headers.map((header, i) => (
+                        <TableHeader
+                          className={`${styles.productiveHeading01} ${styles.text02}`}
+                          {...getHeaderProps({
+                            header,
+                            isSortable: header.isSortable,
+                          })}
+                        >
+                          {header.header}
+                        </TableHeader>
+                      ))}
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {rows.map((row, i) => (
+                      <TableRow {...getRowProps({ row })} key={row.id}>
+                        <TableCell key={row.cells[0].id}>
+                          <Link
+                            style={{ cursor: 'pointer' }}
+                            onClick={() => {
+                              handleFormOpen(row.id, '', tableRows[i].formName);
```

Comment:
Better to use `null` here...

```suggestion
                              handleFormOpen(row.id, null, tableRows[i].formName);
```